### PR TITLE
Fix code format and slice view.

### LINKED ISSUE #15: [Feature] Slice fulfillment showcases
Allow app to present slice mode fulfillment options

### PR DESCRIPTION
**Pull request by RazorBach**
Fix all code format.
Also add slice view into the app. Current slice view is displayed in large mode.

Fixes #15 #35

### CODE REVIEW

File: AppActionVisualizer/app/src/main/java/com/example/appactionvisualizer/databean/AppActionsGenerator.java [hunk 1]
```diff
@@ -25,16 +25,13 @@
   private static AppActionsGenerator single_instance = null;
 
   public static AppActionsGenerator getInstance() {
-    if (single_instance == null)
-      single_instance = new AppActionsGenerator();
+    if (single_instance == null) single_instance = new AppActionsGenerator();
```
Comment (jianjinjian) [inline AppActionVisualizer/app/src/main/java/com/example/appactionvisualizer/databean/AppActionsGenerator.java]: wrap with{} although it's one line

File: AppActionVisualizer/app/src/main/java/com/example/appactionvisualizer/databean/AppActionsGenerator.java [hunk 2]
```diff
@@ -25,16 +25,13 @@
...
     return single_instance;
   }
 
-
   // Parse the data using protobuf api
   public void readFromFile(Context context) {
-    if(!appActions.isEmpty())
-      return;
+    if (!appActions.isEmpty()) return;
```
Comment (jianjinjian) [inline AppActionVisualizer/app/src/main/java/com/example/appactionvisualizer/databean/AppActionsGenerator.java]: here as well, add some log before return

**Review by hilaryhuo — APPROVED** [review]

File: AppActionVisualizer/app/src/main/java/com/example/appactionvisualizer/databean/AppActionsGenerator.java (hunk 1 above)
Comment (RazorBach) [reply AppActionVisualizer/app/src/main/java/com/example/appactionvisualizer/databean/AppActionsGenerator.java]: fixed.

File: AppActionVisualizer/app/src/main/java/com/example/appactionvisualizer/databean/AppActionsGenerator.java (hunk 2 above)
Comment (RazorBach) [reply AppActionVisualizer/app/src/main/java/com/example/appactionvisualizer/databean/AppActionsGenerator.java]: Sure. fixed.

**Comment by RazorBach** [comment]
All comments fixed. Also added some guides. Feel free to merge into master branch.

**Pull request merged**